docs/source/delta-sharing.md

```diff
@@ -0,0 +1,204 @@
+
+Once the provider shares a table with history, the recipient can perform a streaming query on the table.
+When you load a Delta Sharing table as a stream source and use it in a streaming query, the query processes all of the data present in the shared table as well as any new data that arrives after the stream is started.
+
+```scala
+val tablePath = "<profile-file-path>#<share-name>.<schema-name>.<table-name>"
+
+spark.readStream.format("deltaSharing").load(tablePath)
+```
+
+Delta Sharing Streaming supports the following functionalities in the same way as Delta Streaming: [Limit input rate](delta-streaming.md#limit-input-rate), 
+[Ignore updates and deletes](delta-streaming.md#ignore-updates-and-deletes), [Specify initial position](delta-streaming.md#specify-initial-position)
+
+In addition, `maxVersionsPerRpc` is provided to decide how many versions of files are requested from the server in every delta sharing rpc. This is to help
+reduce the per rpc workload and make the delta sharing streaming job more stable. 
+Especially when the streaming resumes from a checkpoint and a lots of new versions are accumulated on the shared table on the server. The default is 100.
+
+.. note:: Trigger.AvailableNow is not supported in Delta Sharing Streaming.
+
+## Delta Format Sharing
+Delta Format Sharing is introduced since delta-sharing-client 1.0 and delta-sharing-spark 3.1, in order to support advanced delta features in Delta Sharing.
+DeletionVectors and ColumnMapping are supported.
+With "Delta Format Sharing", the actions of a shared table is returned in delta format, then the delta spark library is leveraged to read data.
+
+Please do remember to set the spark configurations mentioned in [_](delta-batch.md#sql-support) in order to read shared tables with DeletionVectors and ColumnMapping. 
+
+Batch queries can be performed as is, because it can automatically resolve the responseFormat based on the table features of the shared table.
+An additional option `responseFormat=delta` needs to be set for cdf and streaming queries when reading shared tables with DeletionVectors or ColumnMapping enabled.
+
+```scala
+import org.apache.spark.sql.SparkSession
+
+val spark = SparkSession
+        .builder()
+        .appName("...")
+        .master("...")
+        .config("spark.sql.extensions", "io.delta.sql.DeltaSparkSessionExtension")
+        .config("spark.sql.catalog.spark_catalog", "org.apache.spark.sql.delta.catalog.DeltaCatalog")
+        .getOrCreate()
+
+val tablePath = "<profile-file-path>#<share-name>.<schema-name>.<table-name>"
+
+// Batch query
+spark.read.format("deltaSharing").load(tablePath)
+
+// CDF query
+spark.read.format("deltaSharing")
+  .option("readChangeFeed", "true")
+  .option("responseFormat", "delta")
+  .option("startingVersion", 1)
+  .load(tablePath)
+
+// Streaming query
+spark.readStream.format("deltaSharing").option("responseFormat", "delta").load(tablePath)
```

Comment:
this seems like we are repeating the same thing. Is it possible to update the above sections with additional details on what config to enable to read tables with `responseFormat=delta`?

Reply:
considering this is newly introduced to solve a new problem, I feel it's more clear to keep it separate.